region/norfair/east/Bubble Mountain.json

```diff
@@ -3800,12 +3829,41 @@
       "link": [7, 7],
       "name": "Cacatac and Waver Farm",
       "requires": [
-        {"resetRoom": {"nodes": [6, 7]}},
-        {"partialRefill": {"type": "Energy", "limit": 160}},
-        {"partialRefill": {"type": "Super", "limit": 5}},
-        {"partialRefill": {"type": "Missile", "limit": 6}}
+        {"or": [
+          {"and": [
+            {"resetRoom": {"nodes": [1]}},
+            {"or": [
+              {"and": [
+                "Grapple",
+                {"cycleFrames": 300}
+              ]},
+              {"and": [
+                "SpaceJump",
+                {"cycleFrames": 390}
+              ]},
+              {"and": [
+                "HiJump",
+                "canTrickyJump",
```

Comment:
`canTrickyJump` seems a bit of a strong requirement here? In the Base strat it's a `canPreciseWalljump` to get up to the top-left, though upgrading it a bit can make sense because of how it's expected to be repeated. Maybe `canTrickyWalljump` would be enough? In any case probably it should be consistent across all the other HiJump+walljump farm strat variants that go up here (unless I'm missing something that makes them different?)

Reply:
I thought we put right side wall climb in very hard to avoid king cac.  But actually it is doable with canTrickyWalljump, and with HiJump it is just a carefuljump.  I'll leave it with canTrickyWalljump